### PR TITLE
Update for Twitter using Testarina

### PR DESCRIPTION
**Pull request by dilinisg**

### CODE REVIEW

File: twitter/samples/twitter/twitter_test.bal [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+}
+
+
+function testTweet() {
+        twitter:ClientConnector twitterConnector = init();
+        message response = twitter:ClientConnector.tweet(twitterConnector, "#ballerina");
+        json jsonResponse = messages:getJsonPayload(response);  
+        int status = http:getStatusCode(response);
+	string value = jsons:getString(jsonResponse,"text");
```
Comment (lankavitharana) [inline twitter/samples/twitter/twitter_test.bal]: shall we remove jsons functions from here? see the below example
json tmp = {"sample":{"saman":"name"}};
string val = tmp.sample.saman;

File: twitter/samples/twitter/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	Consumer Key (API Key)
+	Consumer Secret (API Secret)
+	Access Token
+	Access Token Secret
+ 2. Set the above twitter account details in the set_env_vars.sh.
+ 3. Run the set_env_vars.sh and source the file in the terminal.
+ 4. Navigate to $BALLERINA_TOOLS_HOME/bin
+ 5. Run tests in the same terminal as:
+ ./ballerina test ${path_to_test_file_in_connector_repo}/twitter_test.bal
+ 6. Get the Twitter ID which will be printed on the console
```
Comment (lankavitharana) [inline twitter/samples/twitter/README.md]: shall we use 'set_env_vars.sh' to set twitterId as well?

**Comment by lankavitharana** [comment]
Shall we follow [1] when doing commits and pull requests?

[1] - https://chris.beams.io/posts/git-commit/

File: twitter/samples/twitter/README.md (hunk 2 above)
Comment (dilinisg) [inline twitter/samples/twitter/README.md]: We cannot set the Twitter ID because we don't know about it when executing the tests first

**Pull request closed without merging**